dspace-api/src/main/java/org/dspace/discovery/SolrServiceImpl.java

```diff
@@ -2216,4 +2258,29 @@ public FacetYearRange getFacetYearRange(Context context, DSpaceObject scope, Dis
         result.calculateRange(context, filterQueries, scope, this);
         return result;
     }
+
+    @Override
+    public String calculateExtremeValue(Context context, String valueField,
```

Comment:
Is the SolrStats component a better choice here?

Reply:
I'm not sure which SolrStats component you are referring to. Can you share the full class name?

Reply:
I'm talking about the Stats Component. In our SOLR version tt can be used over numeric & date fields and provide direct access to the min and max values
http://localhost:8080/solr/search/select?q=*%3A*&rows=0&wt=json&indent=true&stats=true&stats.field=dc.date.accessioned_dt